_posts/en/newsletters/2024-06-07-newsletter.md

```diff
@@ -0,0 +1,281 @@
+---
+title: 'Bitcoin Optech Newsletter #306'
+permalink: /en/newsletters/2024/06/07/
+name: 2024-06-07-newsletter
+slug: 2024-06-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces an upcoming disclosure of
+vulnerabilities affecting older versions of Bitcoin Core, describes a
+draft BIP for a new version of testnet, summarizes a proposal for
+covenants based on functional encryption, examines an update to the
+proposal for performing 64-bit arithmetic in Bitcoin Script, links to a
+script for validating proof-of-work on signet with the `OP_CAT` opcode, and
+looks at a proposed update to the BIP21 specification of `bitcoin:`
+URIs.  Also included are our regular sections announcing new releases
+and release candidates, plus summaries of notable changes to popular
+Bitcoin infrastructure software.
+
+## News
+
+- **Upcoming disclosure of vulnerabilities affecting old versions of Bitcoin Core:**
+  several members of the Bitcoin Core project [discussed][irc disclose]
+  on IRC a [proposed policy][disclosure policy] for disclosing
+  vulnerabilities that affected older versions of Bitcoin Core.  For
+  low-severity vulnerabilities, the details will be disclosed about two
+  weeks after the first release of a version of Bitcoin Core that
+  eliminates (or satisfactorily mitigates) the vulnerability.  For most
+  other vulnerabilities, the details will be disclosed after the last
+  version of Bitcoin Core affected by the vulnerability reaches
+  end-of-life (which is about a year and a half after it was released).
+  For rare critical vulnerabilities, members of the Bitcoin Core
+  security team will privately discuss the most appropriate disclosure
+  timeline to use.
+
+  After this policy has a chance to be further discussed, it is the
+  intention of the project to begin disclosing vulnerabilities affecting
+  Bitcoin Core 24.x and below.  It is **strongly recommended** that all
+  users and administrators upgrade to Bitcoin Core 25.0 or above within
```

Comment:
Maybe better to recommend 25.2, as 25.0 is over a year old and 25.2 includes a [number of backported bug fixes](https://github.com/bitcoin/bitcoin/compare/v25.0...v25.2).

Reply:
Editing in #1716